src/server/utils/stdin.js

```diff
@@ -0,0 +1,20 @@
+export function stdin() {
+  const stdin = process.stdin;
```

Comment:
I'm not opposed to having a utility for this, but there are already tested stream utilities that handle things like errors a little better:

```js
import { createPromiseFromStreams, createConcatStream } from 'src/utils'
const stdin = await createPromiseFromStreams([
  process.stdin,
  createConcatStream('')
]);
```

Reply:
Thanks, removed in 4e4f3ba7ff737dfd4b6551b542ca2b269ec10f08